### PR TITLE
update article for performance test

### PR DESCRIPTION
**Pull request by wsm12138**

### CODE REVIEW

File: docs/document/content/reference/test/performance-test/sysbench-proxy-norules-test.cn.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
++++
+title = "Sysbench 性能测试"
```
Comment (TeslaCN) [inline docs/document/content/reference/test/performance-test/sysbench-proxy-norules-test.cn.md:2]: Consider enriching the title.

Comment (wsm12138) [reply docs/document/content/reference/test/performance-test/sysbench-proxy-norules-test.cn.md]: > Consider enriching the title.

fixed

File: docs/document/content/reference/test/performance-test/sysbench-proxy-norules-test.cn.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+rules: []
+
+```
+
+## 测试阶段
+
+### 环境准备
+
+```shell
+sysbench oltp_read_write --mysql-host='${DB_IP}' --mysql-port=${DB_PORT} --mysql-user=${USER} --mysql-password='${PASSWD}' --mysql-db=test --tables=10 --table-size=1000000 --report-interval=10 --time=100 --threads=200  cleanup
```
Comment (TeslaCN) [inline docs/document/content/reference/test/performance-test/sysbench-proxy-norules-test.cn.md:85]: Consider removing the single quote.

Comment (wsm12138) [reply docs/document/content/reference/test/performance-test/sysbench-proxy-norules-test.cn.md]: > single quote

fixed

File: docs/document/content/reference/test/performance-test/sysbench-proxy-norules-test.cn.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+...
+[ 120s ] thds: 200 tps: 11731.00 qps: 234638.36 (r/w/o: 164251.67/46924.69/23462.00) lat (ms,95%): 24.38 err/s: 0.00 reconn/s: 0.00
+SQL statistics:
+    queries performed:
+        read:                            19560590                       # 读总数     
+        write:                           5588740                        # 写总数
+        other:                           27943700                       # 其他操作总数 (COMMIT 等)
+        total:                           27943700                       # 全部总数
+    transactions:                        1397185 (11638.59 per sec.)    # 总事务数 ( 每秒事务数 )
+    queries:                             27943700 (232771.76 per sec.)  # 读总数 ( 每秒读次数 )
```
Comment (TeslaCN) [inline docs/document/content/reference/test/performance-test/sysbench-proxy-norules-test.cn.md:116]: Could you double check the comment here?

**Review by TeslaCN — APPROVED** [review]

**Pull request merged**